compiler/rustc_next_trait_solver/src/canonicalizer.rs

```diff
@@ -1,5 +1,6 @@
 use std::cmp::Ordering;
 
+#[cfg(feature = "nightly")]
```

Comment:
🤔 how does that depend on nightly? and why is it only necessary with `nightly` enabled

Reply:
it was probably a mistype or a bad merge that put this line onto the wrong import